components/datetime/src/provider/mod.rs

```diff
@@ -193,5 +193,17 @@ pub mod gregory {
             pub medium: Cow<'static, str>,
             pub short: Cow<'static, str>,
         }
+
+        #[derive(Debug, PartialEq, Clone, Default)]
+        #[cfg_attr(
+            feature = "provider_serde",
+            derive(serde::Serialize, serde::Deserialize)
+        )]
+        pub struct DateTimeFormatsV1 {
```

Comment:
At what point do we need to care about versioning conflicts, and will we have a testing strategy for catching them? I'm assuming for now we're considering this format to be unstable.

Reply:
Don't worry about stability of data structs until we reach v1.